### ISSUE TITLE
Less mess constants in code

### ISSUE THREAD

**Issue by MaksymTaratuta55** [ramsayinnovations/ramsay-site#46]
For example Footer.js: you don't have to create "const data" in the component. And create it at once without an empty object. Also, you can store it in Footer.constants.js for more cleaner component.
Example:
```
const data = {
 [
    {
      title: "Links",
      list: [
        {
          name: "Terms and Conditions",
          webLink: "https://www.termsfeed.com/live/052cf669-4b0d-4d3b-aea4-388ded2de1f8",
        },
        {
          name: "Privacy Policy",
          webLink: "https://www.termsfeed.com/live/54160450-d627-4713-b38b-729c3a56295b",
        },
        {
          name: "Cookies Policy",
          webLink: "https://www.termsfeed.com/live/172ec218-b2c1-43e4-ad23-7ac3c63e6653",
        },
      ]
    },
  ],
  [
    {
      title: "Learn More",
      list: [
        {
          name: "Ramsay Method",
          id: SECTION_IDS.RAMSAY_METHOD,
        },
        {
          name: "Ramsay Advisor",
          id: SECTION_IDS.RAMSAY_ADVISOR,
        },
        {
          name: "Ramsay Commitment",
          id: SECTION_IDS.RAMSAY_COMMITMENT,
        },
      ]
    },
  ]
};
```

**Comment by thecodeclimber** [comment]
@kirti75waytech 

Please work on implementing this.  Please do so for all relevant files.  If you have any questions, please feel free to ask @MaksymTaratuta55 

Thanks.

**Comment by thecodeclimber** [comment]
@kirti75waytech -- please let me know where we are holding with this.

**Comment by kirti75waytech** [comment]
@thecodeclimber I have started working on this.

**Comment by kirti75waytech** [comment]
@MaksymTaratuta55 There are several files where we have constant data. So, as per this ticket, should I move them in the respective constant.js file. Or should I move all these type of constants in a single constant.js file?
For your reference I am providing you the list where we have such constant data :
- Footer.jsx,
- Header.jsx,
- RamsayAdvisor.jsx,
- RamsayCommitment.jsx,
- RamsayMethod.jsx,
- Testimonial.jsx

In the present scenario I have created two different constant.js files for the header & footer, and separated the code.
-components
   -layouts
      -main
          *header
             -Header.jsx
             -Header.constant.js
          *footer
             -Footer.jsx
             -Footer.constant.js
         -MainLayout.jsx

Please suggest me, which type of structure should I follow for it?

**Comment by MaksymTaratuta55** [comment]
@kirti75waytech, no, I think a single constant.js file makes it difficult to read and find data when you need it.
You can create single constant.js in utils but use it for other purposes. These constants are component-level data, you need it only in current components. So I recommend to create it in the folder with component `./header/Header.constants.js` or just `./header/constants.js`

**Comment by kirti75waytech** [comment]
Okay @MaksymTaratuta55 Noted this point.

**Comment by kirti75waytech** [comment]
@MaksymTaratuta55 @thecodeclimber I have resolved this issue and created the PR #71 for this issue. Under this issue I have made constant files for Testimonial, RamsayMethod, RamsayCommitment, RamsayAdvisor, Footer, Header and ContactUs component. Please review it and let me know if any changes are required.
Thanks
